### DISCUSSION TITLE
NextAuth implementation on Next.js with App routing

### DISCUSSION THREAD

**Discussion by pavlo-js** [nextauthjs/next-auth#8321] [Help]
Hi all.
Now, I'm developing an online tutoring web app using Next.js.
I'm going to use NextAuth on my app with MySQL DB and email provider.
Is there anyone who can provide me a good example of it?
I'm newbie at JS and next.js. so I checked a lot of matters describing NextAuth with page routing.
But I couldn't convert it into my latest Next.js project.

Thanks in advance. :)

**Comment by mr-chandan** [comment]
The documentation is pretty good but if are looking to wrap nextauth in next js 13 the app directory https://youtu.be/57099pou7iE this videos will help you
